zipline/assets/assets.py

```diff
@@ -996,8 +996,13 @@ def _get_root_symbol_exchange(self, root_symbol):
 
         fields = (fc_cols.exchange,)
 
-        return sa.select(fields).where(
-            fc_cols.root_symbol == root_symbol).execute().fetchone()[0]
+        exchange_tuple = sa.select(fields).where(
+            fc_cols.root_symbol == root_symbol).execute().fetchone()
```

Comment:
Good point, afaik there would never be more than one match, so `scalar()` is better here.